src/utils.js

```diff
@@ -67,10 +67,14 @@ export function getDefaultFormState(schema) {
     return schema.default;
   }
   if (schema.type === "object") {
-    return Object.keys(schema.properties).reduce((acc, key) => {
-      acc[key] = getDefaultFormState(schema.properties[key]);
-      return acc;
-    }, {});
+    if (schema.properties) {
+      return Object.keys(schema.properties).reduce((acc, key) => {
+        acc[key] = getDefaultFormState(schema.properties[key]);
+        return acc;
+      }, {});
+    } else {
+      return {}
+    }
```

Comment:
Smart, that's a much better approach. I'll kill this PR and create a new one with only that change, keep the git history neater.